types/pdfmake/index.d.ts

```diff
@@ -171,16 +181,24 @@ declare module 'pdfmake/build/pdfmake' {
     }
 
     interface TDocumentDefinitions {
-        info?: TDocumentInformation;
+        background?: (currentPage?: any, pageSize?: any) => any;
         compress?: boolean;
-        header?: TDocumentHeaderFooterFunction;
-        footer?: TDocumentHeaderFooterFunction;
         content: string | Content;
-        styles?: Style;
-        pageSize?: PageSize;
-        pageOrientation?: PageOrientation;
-        pageMargins?: Margins;
         defaultStyle?: Style;
+        footer?: TDocumentHeaderFooterFunction;
+        header?: TDocumentHeaderFooterFunction;
+        images?: Record<string, string>;
```

Comment:
Can you explain what `Record<string, string>` does here? I assume you take this from [here](https://pdfmake.github.io/docs/document-definition-object/images/). In my opinion, the following is easier to read (because I assume some people would have to look up `Record`):

```
images?: { [key: string]: string };
```

Reply:
Do you use the `Record` type in your daily work for types like this? Just out of curiosity :D

Reply:
I do use Record, but it is exactly the same as far as I understood as the `{ [key: string]: string }`